recipe/migrations/cuda128.yaml

```diff
@@ -0,0 +1,61 @@
+migrator_ts: 1738229377
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: false
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  ordering:
+    cuda_compiler:
+      - None
+      - nvcc
+      - cuda-nvcc
+    cuda_compiler_version:
+      - None
+      - 11.8
+      - 12.4
+      - 12.6
+      - 12.8
```

Comment:
Past migrators have also contained this field. However they included a lot more detail to do the different Docker images we provided for CUDA versions and other details

Since we are at a point where CUDA 12.x reuses a lot of the same content from our global matrix and doesn't change it, found this could be greatly reduced compared to past migrations

Reply:
To test whether the ordering stuff works, I removed `operation: key_add` as suggested by Isuru, and tried
```yaml
  ordering:
    cuda_compiler:
      - [...]
    cuda_compiler_version:
      - None
      - 12.4
      - 12.6
      - 11.8
      - 12.8
``` 
and
```yaml
  ordering:
    cuda_compiler:
      - [...]
    cuda_compiler_version:
      - 12.4
      - 12.6
      - None
      - 11.8
      - 12.8
``` 
on the faiss feedstock (not that many feedstocks I'm aware of which still build 11.8). From what I can tell, `None` is not handled correctly - in both cases, the resulting jobs are for CUDA 11.8, 12.6 & 12.8, and the CPU job gets removed.